[[1]]
 [1] "glmmTMB"    "torch"      "FINN"       "data.table" "stats"     
 [6] "graphics"   "grDevices"  "utils"      "datasets"   "methods"   
[11] "base"      

[[2]]
 [1] "glmmTMB"    "torch"      "FINN"       "data.table" "stats"     
 [6] "graphics"   "grDevices"  "utils"      "datasets"   "methods"   
[11] "base"      

[[3]]
 [1] "glmmTMB"    "torch"      "FINN"       "data.table" "stats"     
 [6] "graphics"   "grDevices"  "utils"      "datasets"   "methods"   
[11] "base"      

[[4]]
 [1] "glmmTMB"    "torch"      "FINN"       "data.table" "stats"     
 [6] "graphics"   "grDevices"  "utils"      "datasets"   "methods"   
[11] "base"      

Error in checkForRemoteErrors(val) : 
  3 nodes produced errors; first error: NA/NaN argument
Calls: <Anonymous> ... clusterApply -> staticClusterApply -> checkForRemoteErrors
Execution halted
